src/plugins/orangehrmAdminPlugin/Api/ModulesAPI.php

```diff
@@ -65,6 +66,7 @@ class ModulesAPI extends Endpoint implements CrudEndpoint
         self::PARAMETER_MAINTENANCE => false,
         self::PARAMETER_MOBILE => false,
         self::PARAMETER_DIRECTORY => false,
+        self::PARAMETER_CLAIM => false,
```

Comment:
I feel this and all other modules' default values should go as `true`. but I don't this we can change this now as its BC break